src/ParallelAlgorithms/Amr/Projectors/Protocols/Mutators.hpp

```diff
@@ -0,0 +1,31 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+#include "ParallelAlgorithms/Amr/Projectors/Protocols/Mutator.hpp"
+
+namespace amr::protocols {
+/// \brief Compile-time information for AMR mutators
+///
+/// A class conforming to this protocol is placed in the metavariables to
+/// provide the list of AMR mutators that will be applied by:
+/// - amr::Actions::InitializeChild and amr::Actions::InitializeParent in order
+///   to initialize data on newly created elements.
+/// - amr::Actions::AdjustDomain in order to update data on existing elements in
+///   case their Mesh or neighbors have changed.
+///
+/// Here is an example for a class conforming to this protocol:
+///
+/// \snippet Projectors/Test_Protocol.cpp amr_mutators
+struct Mutators {
```

Comment:
How about we name this `AmrMetavariables` and in the metavars it's a section like this:

```cpp
struct amr : tt::ConformsTo<amr::protocols::AmrMetavariables> {
  using mutators = tmpl::list<...>;
};
```

Then this would be a place to put more metavars if needed and a way to have documented "sections" in the metavars.